service.py

```diff
@@ -37,9 +38,69 @@ def autostart():
         else:
             cleaner.monitor.waitForAbort(service_sleep)
 
-    log("Abort requested. Terminating.")
+    debug("Abort requested. Terminating.")
     return
 
 
+    
+class Main:
```

Comment:
This an old-style class definition. You should use ``class Main()`` instead.

Quick question: is it really necessary to create this class just to create the ``MyPlayer`` object? I have not used the ``xbmc.Monitor`` class before.